pavelib/pep8.py

```diff
@@ -0,0 +1,27 @@
+from __future__ import print_function
+
+from pavelib.utils.envs import Env
+from pavelib.pylint import get_clean_report_directory
+from pavelib.utils.cmd import shell
+
+
+def pep8(services=None):
+    services = services or []
+    services = ' '.join(services)
+    Env.METRICS_DIR.makedirs_p()
+    report_dir = get_clean_report_directory('pep8')
+    report_file = "{report_dir}/pep8.report".format(
+        report_dir=report_dir,
+    )
+    shell(
+        'pep8',
+        services,
```

Comment:
I have tested this. This was one of my outstanding annotations, so I'm glad that you pointed it out; this behaves slightly differently than before.

Previously, separate invocations of `pep8` were made for each service. So you'd have `pep8 lms`, `pep8 cms`, `pep8 common`, and so forth. Since we were outputting to a single log file, it was necessary to append (as well as explicitly wiping the file at the onset).

Now, since we only need a single output file, we can invoke `pep8` just a single time, passing in all services as a space-delimited list `pep8 lms cms common`. An empty services list, the default case, amounts to a bare `pep8` invocation, which should scan the entire CWD.